Object[] objects
int32 size